internal/api/handlers/operations_handler_test.go

```diff
@@ -454,6 +454,124 @@ func TestCancelOperation(t *testing.T) {
 	})
 }
 
+func TestGetOperation(t *testing.T) {
+	dates := []time.Time{
+		time.Time{}.AddDate(2020, 0, 0),
+		time.Time{}.AddDate(2020, 1, 0),
+		time.Time{}.AddDate(2020, 2, 0),
+	}
+
+	type tableTestInput struct {
+		Operation *operation.Operation
```

Comment:
This guy over here is mock preparation, right? it seems to me that this Operation field is not actually input for the test but rather some kind of mock preparation. WDYT?